project/SparkBuild.scala

```diff
@@ -1059,9 +1059,25 @@ object KubernetesIntegrationTests {
 object DependencyOverrides {
   lazy val guavaVersion = sys.props.get("guava.version").getOrElse("33.1.0-jre")
   lazy val settings = Seq(
-    dependencyOverrides += "com.google.guava" % "guava" % guavaVersion,
-    dependencyOverrides += "jline" % "jline" % "2.14.6",
-    dependencyOverrides += "org.apache.avro" % "avro" % "1.11.3")
+    dependencyOverrides ++= {
```

Comment:
Do you want to keep this? Although the issue was fixed when `netty` upgraded to version `1.1.114`, similar problems may still occur next time when other libraries depend on `different netty` versions.